### DISCUSSION TITLE
Use real image

### DISCUSSION THREAD

**Discussion by tretts** [lllyasviel/Fooocus#2308] [Q&A] [answered]
Hi everyone, i tried different combination to modify an image, but i'm not able to stick a real face in the new image, probably i'm use the wrong method. I tried "image prompt" with "face swap" but result sucks.
Do you any better solution?

Thanks

**Comment by eddyizm** [comment]
This is a common request but without knowing your exact settings, models, values, and original images and output images it's hard to give you any advice or support.

**Comment by tretts** [comment]
Well i'm quite new so i'm not sure which kind of settings you need to know, i'm using "run-realism" and basic settings just to understand how it works but i'm not that sure, the original image is a selfie and i would use to modify the background merging it to obtain a photo profile.
Let me know if it's enough, thanks

**Reply by eddyizm** [reply]
So far, I get you are running the realism bat file, so I know which model you are using. Can you explain how you are doing this part
"the original image is a selfie and i would use to modify the background merging it to obtain a photo profile."

Screenshots would be helpful to start and see how you are using image prompt and face swap. There are advanced settings as well and we can point you in a better direction if we understand the flow you are currently trying.

**Reply by tretts** [reply]
Well i'm not that sure using the correct bat file, but i suppose that i have to use image prompt and faceswap to achive the result. Face chenges everytime and i don't how to keep it and change the rest.
Attached an example.
![image](https://github.com/lllyasviel/Fooocus/assets/3592231/be05334f-3528-4beb-8345-a15cbd1ed94e)

**Reply by SGThirkell** [reply, marked as answer]
I would put your original image in the inpaint section. Mask all the background but not you. Describe a prompt you want the background to be (eg portrait of a man on a pale blue background with neural net connections, blue glow) and tick the Left Right and Bottom Outpaint direction to grow the finished image.
